web/src/routes/(user)/people/+page.svelte

```diff
@@ -423,7 +430,10 @@
   </FullScreenModal>
 {/if}
 
-<UserPageLayout title="People" description={countTotalPeople === 0 ? undefined : `(${countTotalPeople.toString()})`}>
+<UserPageLayout
+  title="People"
+  description={countVisiblePeople === 0 ? undefined : `(${countVisiblePeople.toLocaleString($locale)})`}
```

Comment:
Why are you doing the string building like this? Just put {count} inbetween the curly braces and put the parens outside of them to like ({count}). I think that would work fine. In general, I have found it rare that es6 template strings are needed inside of a svelte component html.

Reply:
Sorry, I don't get how you can concatenate strings without using es6 template strings, do you prefer something like this ? ```ts
'(' + countVisiblePeople.toLocaleString($locale) + ')'

Reply:
Since it's svelte `{}` already escapes strings.
So what you can do is
```ts
description={countVisiblePeople === 0 && "({countVisiblePeople.toLocaleString($locale)})"}
```
I *think*.
Note that I've also changed `countVisiblePeople === 0 ? undefined : ...` to `countVisiblePeople === 0 && ...`

Reply:
Sorry you're right. This should:
```ts
description="{countVisiblePeople === 0 && `(${countVisiblePeople.toLocaleString($locale)})`}"
```

Reply:
Ah, I didn't realize this was in an attribute value, so this should be fine. It seems you fixed the other case though.

Reply:
Sorry I was replying to @danieldietzler